### PR TITLE
Update hls.js to v0.13.0

### PR DESCRIPTION
**Pull request by olafcm**
hls.js updated to v0.13.0, file map corrected

# Pull request template for new library

Pull request for issue: #
Related issue(s): https://github.com/cdnjs/cdnjs/issues/13645

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required):
* Official website (optional, not the repository):
* NPM package url (optional):
* License and its reference:
* GitHub / Bitbucket popularity (required):
  * Count of watchers:
  * Count of stars:
  * Count of forks:
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [ ] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/hls.js/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hls.js",
-  "version": "0.12.4",
+  "version": "0.13.0",
```
Comment (MattIPv4) [inline ajax/libs/hls.js/package.json]: Please include this version and any missing versions in this PR.

Comment (olafcm) [reply ajax/libs/hls.js/package.json]: 0.12.4 is already on cdnjs and 0.13.0 is (apart from some beta's) the only missing version.
So no changes required or do I oversee something?
Thanks :)

Comment (olafcm) [reply ajax/libs/hls.js/package.json]: I have also included the missing 0.13.0 files now.
Please let me know if this is OK :)

Comment (olafcm) [reply ajax/libs/hls.js/package.json]: Is it ready to be merged now?
Would be nice 😊

**Comment by olafcm** [comment]
@MattIPv4 is this pull request ready for merge now? Thanks for checking :-)

**Comment by xtuc** [comment]
`v0.13.1` has been added by the bot. Is this needed?

**Comment by MattIPv4** [comment]
I think this can be closed. 0.13.0-beta.1 & 0.13.1 are in the repo now.

**Comment by olafcm** [comment]
Yes all good :-D

**Pull request closed without merging**